controllers/storagecluster/external_resources.go

```diff
@@ -390,22 +391,17 @@ func (r *StorageClusterReconciler) createExternalStorageClusterResources(instanc
 					return err
 				}
 			} else if d.Name == cephRgwStorageClassName {
-				rgwEndpoint := d.Data[externalCephRgwEndpointKey]
-				if err := checkEndpointReachable(rgwEndpoint, 5*time.Second); err != nil {
-					r.Log.Error(err, "RGW endpoint is not reachable.", "RGWEndpoint", rgwEndpoint)
-					return err
-				}
-				extCephObjectStores, err = r.newExternalCephObjectStoreInstances(instance, rgwEndpoint)
-				if err != nil {
-					return err
-				}
```

Comment:
If you log the error and continue instead of returning an error here, wouldn't that be enough?
Later you can just check whether `extCephObjectStores`  and `rgwEndpoint` are nil.

Reply:
Later then we have to check them separately and log them separately with separate errors,
 and error out separately to get the operator reconciled.
 So that would be a double check I believe....